microcosm_eventsource/ddl/delete_event_type.create.ddl

```diff
@@ -0,0 +1,39 @@
+/*
+ * delete_event_type(table_name, old_event_type, model_id_row_name)
+ *
+ * Should be only used by data migrations scripts.
+ * Delete events with old_event_type and remove it from states,
+ * Must pass model_id_row_name - the row name that links to the parent model
+ * (Example: for company_event table, the model_id_row_name should be company_id)
+ *
+ * Details:
+ * - Updates parent_id of relevant child events
+ * - Updates the state column (Ensures that the state is sorted and has unique events)
+ * - Does not promise valid events or transitions.
+ * - Cannot be used if model.__unique_parent__ is set to False (missing table_names_parent_id_key constraint),
+ */
+CREATE OR REPLACE FUNCTION delete_event_type(
+    table_name regclass,
```

Comment:
There is no easy way to pass tables to functions in psql.
So we're passing strings and using `EXECUTE format(...)`
For migrations - it's safe enough.